common-libs/internal-api-clients/src/services/KeyStorageApiService.ts

```diff
@@ -82,6 +82,11 @@ export default class KeyStorageApiService {
     return this.client.AdminLogOutUser({ authorization: accessToken })
   }
 
+  async adminGetUserInfo({ accessToken }: { accessToken: string }) {
+    const result = await this.client.AdminGetUserInfo({ authorization: accessToken })
```

Comment:
Just FYI
it is better to use auto-generated types here next time:
```
async adminGetUserInfo(params: GetParams<typeof clientMethods.AdminGetUserInfo>) {
    const result = await this.client.AdminGetUserInfo(params)
```

Reply:
sure, will change it next time.